app/lib/bk/compat/parsers/harness.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require_relative '../translator'
+require_relative '../pipeline'
+require_relative '../pipeline/step'
+require_relative './harness/steps/run'
+require_relative './harness/steps/background'
+
+module BK
+  module Compat
+    # Harness translation scaffolding
+    class Harness
+      include StepTranslator
+      require 'yaml'
+
+      def self.name
+        'Harness'
+      end
+
+      def self.option
+        'harness'
+      end
+
+      def self.matches?(text)
+        # sorting is important
+        config = YAML.safe_load(text, aliases: true)
+        mandatory_keys = %w[pipeline].freeze
+
+        if config.is_a?(String)
+          false
+        elsif mandatory_keys & config.keys == mandatory_keys
+          config['pipeline'].include?('stages')
+        else
```

Comment:
Rubocop not liking the `else` here in CI

Reply:
Made me think how to rewrite it to avoid the complaint, but I think I got it